### PR TITLE
Formatter

### PR DESCRIPTION
**Pull request by JuanSeZ**

### CODE REVIEW

**Comment by JuanSeZ** [comment]
Tests are missing rn

**Comment by JuanSeZ** [comment]
Added a few more tests, but we're missing the parsers refactor on the new ast changes for the operations

File: formatter/src/main/kotlin/formatter/DeclarationNodeFormatter.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                append(
+                    formatterMap[declarationNode.expression!!::class]?.format(declarationNode.expression!!, rule, formatterMap)
+                        ?: throw IllegalArgumentException("No formatter found for ${declarationNode.expression!!::class}"),
+                )
+            }
+        }
+    }
+}
+
+private fun humanize(input: String): String {
```
Comment (MatiBrego) [inline formatter/src/main/kotlin/formatter/DeclarationNodeFormatter.kt:32]: I think I don't quite understand this function. Is it making the variable type's first letter upper case?

If so, I don't think this is the formatter's responsibility.

**Review by MatiBrego — APPROVED** [review]
LGTM, great job!

**Pull request merged**